test/ae_mdw_web/controllers/name_controller_test.exs

```diff
@@ -1929,6 +1929,197 @@ defmodule AeMdwWeb.NameControllerTest do
     end
   end
 
+  describe "name_transfers" do
+    test "it returns all of the name claims in backward order", %{conn: conn, store: store} do
```

Comment:
here in the description and on the response data would be name transfers

Reply:
Done